docs/proposals/Avatar/README.md

```diff
@@ -0,0 +1,78 @@
+# Avatar
+
+An `<Avatar />` component will be used to display an identifier (image or
+initials) for a user.
+
+Rebecca Li has done some
+[great research](https://jobber.atlassian.net/browse/JOB-19028) on the usage of
+avatars in Jobber, as well as the potential usage of colors within avatars.
+
+## Interface
+
+```
+<Avatar
+    image="ImageUrl"
+    name="Eddy Sims"
+    initials="ES"
+    color="#8f2d56" | color="var(--color-red)"
+    size="large" | size={32}
+/>
+```
+
+## Props Table
+
+| name     | type   | default  | description                               |
+| -------- | ------ | -------- | ----------------------------------------- |
+| image    | String | -        | The avatar image to be use                |
+| name     | String | -        | A users name to be used for accessibility |
+| initials | String | -        | Initials to show if the image is null     |
+| color    | String | greyBlue | The border color that represents the user |
+| size     | String | small    | Size of the avatar - 'small'              | 'large' |
+
+## Display
+
+An `<Avatar />` component would have a heiarchy of display options
+
+1. Display the image used in the `image` prop.
+2. If `image` is null, display the `initials` prop.
+   - Initials should fit within the avatar and scale with the size.
+   - If initials contain 3 letters, the font size should scale to proplerly fit
+     this
+3. If `image` and `initials` are null. Display a fallback icon
+   - We could use `person` from https://atlantis.getjobber.com/components/icon
+
+## Color
+
+Can be used to represent the border color of the avatar if an image is set, or
+the background color of the avatar if the image is not set.
+
+One thing to note is that for accessibility purposes, color selection could
+cause issues with color contrast.
```

Comment:
Currently calendar colours receive a class that adjusts text colour to account for background color, but this is all hard-coded (eg `bg_purple` sets the text to be white because we know the background is purple). Not saying we need to solve at this component level, but could we do [something automated](https://contrast-ratio.com/) when a calendar colour is set to calculate contrast and choose black or white text accordingly? Would also be an interesting tool to have for the color picker component.

Reply:
This is something we are considering, but maybe not at the first pass of this component. We do however need something to make sure contrast is appropriate as soon as we allow custom colors.